src/NavBar.js

```diff
@@ -324,29 +325,44 @@ class NavBar extends React.Component {
 
   renderTitle(childState, index:number) {
     const title = this.props.getTitle ? this.props.getTitle(childState) : childState.title;
+    const titleNumberOfLines = this.props.titleNumberOfLines;
+    const style = [
+      styles.title,
+      this.props.titleStyle,
+      this.props.navigationState.titleStyle,
+      childState.titleStyle,
+      {
+        opacity: this.props.position.interpolate({
+          inputRange: [index - 1, index, index + 1],
+          outputRange: [0, 1, 0],
+        }),
+        left: this.props.position.interpolate({
+          inputRange: [index - 1, index + 1],
+          outputRange: [200, -200],
+        }),
+        right: this.props.position.interpolate({
+          inputRange: [index - 1, index + 1],
+          outputRange: [-200, 200],
+        }),
+      },
+    ];
+
+    if (Number.isInteger(titleNumberOfLines) && titleNumberOfLines >= 1) {
```

Comment:
I'm not sure how you can add a conditional attribute in jsx, so I've added a check for the `titleNumberOfLines` prop that will return the `<Animated.Text/>` component with the `numberOflines` attribute.

I would love to know if there is a better way to handle this case as now there is a lot of extra code just for a single optional attribute.